datayoga-py/README.md

```diff
@@ -124,14 +124,18 @@ DataYoga supports both SQL and [JMESPath](https://jmespath.org/) expressions. JM
+| `left`               | Returns a specified number of characters from the start of a given text string                                                   | Input: `{"greeting": "hello world!"}` <br /> Expression: `` left(greeting, `5`) `` <br /> Output: `hello`                                         |
+| `lower`              | Converts all uppercase characters in a string into lowercase characters                                                          | Input: `{"fname": "John"}` <br /> Expression: `lower(fname)` <br /> Output: `john`                                                                |
+| `mid `               | Returns a specified number of characters from the middle of a given text string                                                  | Input: `{"greeting": "hello world!"}` <br /> Expression: `` mid(greeting, `4`, `3`) `` <br /> Output: `o w`                                       |
+| `replace`            | Replaces all the occurrences of a substring with a new one                                                                       | Input: `{"sentence": "one four three four!"}` <br /> Expression: `replace(sentence, 'four', 'two')` <br /> Output: `one two three two!`           |
+| `right`              | Returns a specified number of characters from the end of a given text string                                                     | Input: `{"greeting": "hello world!"}` <br /> Expression: `` right(greeting, `6`) `` <br /> Output: `world!`                                       |
+| `split`              | Splits a string into a list of strings after breaking the given string by the specified delimiter (comma by default)             | Input: `{"departments": "finance,hr,r&d"}` <br /> Expression: `split(departments)` <br /> Output: `['finance', 'hr', 'r&d']`                      | Default delimiter is comma - a different delimiter can be passed to the function as the second argument, for example: `split(departments, ';')` |
+| `upper`              | Converts all lowercase characters in a string into uppercase characters                                                          | Input: `{"fname": "john"}` <br /> Expression: `upper(fname)` <br /> Output: `JOHN`                                                                |
+| `uuid`               | Generates random uuid4 and returns it as string in standard format.                                                              | Input: None <br /> Expression: `uuid()` <br /> Output: `3264b35c-ff5d-44a8-8bc7-9be409dac2b7`                                                     |
```

Comment:
returns it as string --> returns it as a string

Reply:
✅